geopandas/tools/crs.py

```diff
@@ -0,0 +1,38 @@
+import pyproj
+import re
+import os
+import fiona.crs
+
+
+def explicit_crs_from_epsg(crs=None, epsg=None):
+    """
+    Gets full/explicit CRS from EPSG code provided.
+
+    Parameters
+    ----------
+     crs : dict or string, default None
+         An existing crs dict or string with the 'init' key specifying an EPSG code
+     epsg : string or int, default None
+        The EPSG code to lookup
+    """
+
+    if epsg is None:
+        if crs is not None:
+            if isinstance(crs, str):
+                crs = fiona.crs.from_string(crs)
+            if 'init' in crs and crs['init'].lower().startswith('epsg:'):
+                epsg = crs['init'].split(':')[1]
+
```

Comment:
This section seems potentially worthy of splitting into its own function, ala https://github.com/geopandas/geopandas/pull/457/files#diff-3f5a48cd509b85c0e84ca73d1e2a8151R55

Reply:
I disagree as this is simply filling the epsg parameter, not a separate utility function.

Reply:
The referenced function in the other PR is doing the same things as these lines here. I think your approach here is probably a little better, and would be nice to be able to re-use it instead of duplicating it.

Reply:
Ah! I understand now. I'll split this out as it's own function in the same file...perhaps called `epsg_from_crs`? Perhaps with room for improvement later for when `crs` does not contain the `init` parameter (as is often the case with epsg 4326).